docs/404.html

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
do we need this?

Reply:
It believe it's generated by default with `jekyll new`. Seems ok to me.